Example/PaymentSheet Example/PaymentSheetUITest/EmbeddedUITest.swift

```diff
@@ -382,8 +382,8 @@ class EmbeddedUITests: PaymentSheetUITestCase {
         XCTAssertEqual(
             events,
             ["mc_embedded_paymentoption_savedpm_select",
-             "mc_carousel_payment_method_tapped", "mc_open_edit_screen", "mc_embedded_paymentoption_removed",
-             "mc_carousel_payment_method_tapped", "mc_open_edit_screen", "mc_embedded_paymentoption_removed",
+             "mc_open_edit_screen", "mc_embedded_paymentoption_removed",
+             "mc_open_edit_screen", "mc_embedded_paymentoption_removed",
```

Comment:
There's no payment methods being tapped here! We were incorrectly sending the 'tapped' analytic when you delete a saved PM and the selected saved PM changes. 

That doesn't match vertical mode behavior but also doesn't match the name of the analytic since there's no tap happening.

cc @wooj-stripe